src/content/1.7/modules/concepts/_index.md

```diff
@@ -5,3 +5,15 @@ chapter: true
 ---
 
 # Extension concepts
+
+## Symfony extension concepts
+
+PrestaShop modules acts as Symfony bundles, extension points are added continuously in each minor version:
+
+| Version  | Symfony features                                     |
+|----------|------------------------------------------------------|
+| 1.7.3    | Twig templates inheritance and Services declaration  |
```

Comment:
Are you sure service declaration is part of 1.7.3? :thinking:

Reply:
wow ! you won't be certified if you don't know that @sarjon => https://github.com/PrestaShop/PrestaShop/commit/4efccbad76b6aa1b0c7e13ce08da2a82cf92223a

You can believe me on these subjects: this was my project :)

Reply:
@sarjon is right to ask for confirmation 😉even if it was your project, you can make a mistake about it

This is why there are code reviews 😉

Reply:
it's the day where you guys try to challenge me xD, you're hard I mean... it's only Monday 🤡